src/StandbyPool/StandbyPool/ChangeLog.md

```diff
@@ -19,6 +19,13 @@
 -->
 ## Upcoming Release
 
+## Version 0.3.0
+
+* Updated existing Cmdlets
+  - Get-AzStandbyVMPoolStatus added StandbyPool Health and StandbyPool Prediction information in the response.
+  - Get-AzStandbyContainerGroupPoolStatus added StandbyPool Health and StandbyPool Prediction information in the response.
+  - New-AzStandbyContainerGroupPool added the new parameter -Zones.
```

Comment:
Do not forget to change **Zones** to **Zone** after you change the parameter name to **Zone**